internal/bminventory/inventory_test.go

```diff
@@ -10979,31 +10996,6 @@ var _ = Describe("TestRegisterCluster", func() {
 		Expect(reply).Should(BeAssignableToTypeOf(installer.NewRegisterClusterCreated()))
 	})
 
-	It("SchedulableMasters default value", func() {
-		mockClusterRegisterSuccess(bm, true)
-		mockAMSSubscription(ctx)
-
-		reply := bm.RegisterCluster(ctx, installer.RegisterClusterParams{
-			NewClusterParams: getDefaultClusterCreateParams(),
-		})
-		Expect(reply).Should(BeAssignableToTypeOf(installer.NewRegisterClusterCreated()))
-		actual := reply.(*installer.RegisterClusterCreated)
-		Expect(actual.Payload.SchedulableMasters).To(Equal(swag.Bool(false)))
-	})
-
-	It("SchedulableMasters non default value", func() {
```

Comment:
This should still work, no?

Reply:
Actually, the "SchedulableMasters non default value" test doesn't work because the GetActualSchedulableMasters is now mocked and it returns always 'false'